mods/voidmulti.sc2mod/base.sc2data/gamedata/validatordata.xml

```diff
@@ -556,6 +562,10 @@
         <Value value="TempestDisruptionSphereAttackMissile"/>
         <Find value="0"/>
     </CValidatorUnitType>
+    <CValidatorUnitCompareBehaviorCount id="NotHaveBatteryCooldownBehavior">
+        <WhichUnit Value="Caster"/>
+        <Behavior value="BatteryAcquireTargetCooldown"/>
```

Comment:
Fixed an issue where repeatedly issuing a Stop command could increase the shield regen rate of a Shield Battery's target

New requirement for the auto-cast. A behavior now indicates the cooldown